src/shared/actions/settings.js

```diff
@@ -386,6 +377,20 @@ export function getCurrencyData(currency, withAlerts = false) {
                         ),
                     );
                 }
+            })
+            .catch((error) => {
+                console.log('Error', error);
```

Comment:
```suggestion
                // eslint-disable-next-line no-console
                console.log('Error', error);
```

Reply:
Ah this wasn't meant to be committed. Removed in https://github.com/iotaledger/trinity-wallet/pull/2013/commits/b4d89a20b334feeb4c6b4dbe3a18d13eefd8bcf7.